packages/graphql/tests/integration/issues/2670.int.test.ts

```diff
@@ -0,0 +1,594 @@
+    const genreName1 = "Action";
+    const genreName2 = "Horror";
+    const intValue1 = 1;
+    const intValue2 = 101;
+    const intValue3 = 983;
+    const intValue4 = 0;
+    const intValue5 = 42;
+    const genre2AverageTitleLength = (movieTitle4.length + movieTitle2.length) / 2;
+
+    beforeAll(async () => {
+        neo4j = new Neo4j();
+        driver = await neo4j.getDriver();
+    });
+
+    beforeEach(async () => {
+        session = await neo4j.getSession();
+
+        movieType = generateUniqueType("Movie");
+        genreType = generateUniqueType("Genre");
+        seriesType = generateUniqueType("Series");
+        inGenreInterface = generateUniqueType("InGenre");
+
+        const typeDefs = `
+            type ${movieType.name} {
+                title: String
+                genres: [${genreType.name}!]! @relationship(type: "IN_GENRE", direction: OUT, properties: "${inGenreInterface.name}")
+            }
+
+            type ${genreType.name} {
+                name: String
+                movies: [${movieType.name}!]! @relationship(type: "IN_GENRE", direction: IN, properties: "${inGenreInterface.name}")
+                series: [${seriesType.name}!]! @relationship(type: "IN_GENRE", direction: IN, properties: "${inGenreInterface.name}")
+            }
+
+            type ${seriesType} {
+                name: String!
+                genres: [${genreType.name}!]! @relationship(type: "IN_GENRE", direction: OUT, properties: "${inGenreInterface.name}")
+            }
+
+            interface ${inGenreInterface.name} {
+                intValue: Int!
+            }
+        `;
+
+        neoSchema = new Neo4jGraphQL({
+            typeDefs,
+            driver,
+        });
+
+        await session.run(`
+            CREATE (m1:${movieType.name} { title: "${movieTitle1}" })-[:IN_GENRE { intValue: ${intValue1} }]->(g1:${genreType.name} { name: "${genreName1}" })
+            CREATE (m2:${movieType.name} { title: "${movieTitle2}" })-[:IN_GENRE { intValue: ${intValue2} }]->(g1)
+            CREATE (m3:${movieType.name} { title: "${movieTitle3}" })-[:IN_GENRE { intValue: ${intValue3} }]->(g1)
+            CREATE (m2)-[:IN_GENRE { intValue: ${intValue4} }]->(g2:${genreType.name} { name: "${genreName2}" })
+            CREATE (m4:${movieType.name} { title: "${movieTitle4}" })-[:IN_GENRE { intValue: ${intValue5} }]->(g2)
+            CREATE (m5:${movieType.name} { title: "${movieTitle5}" })
+            CREATE (s1:${seriesType.name} { name: "${seriesName1}" })-[:IN_GENRE { intValue: ${intValue1} }]->(g2)
+            CREATE (s2:${seriesType.name} { name: "${seriesName2}" })-[:IN_GENRE { intValue: ${intValue2} }]->(g2)
+        `);
+    });
+
+    afterEach(async () => {
+        await cleanNodes(session, [movieType, genreType, seriesType]);
+        await session.close();
+    });
+
+    afterAll(async () => {
+        await driver.close();
+    });
+    test("should not throw error", async () => {
```

Comment:
Some more context in the test name would be nice, something like `should not thrown error on nestedAggregate filter in connections"

Reply:
Also, is this test relevant? other tests already check that errors are falsy right? or Am I missing something?

Reply:
Yeah tbh this can probably just be removed. This was just the initial test I was using when working on this